prisma/schema.prisma

```diff
@@ -0,0 +1,62 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
```

Comment:
Using prisma has been a pretty interesting journey. Their tooling (especially the editor tooling to write these schema files) is top notch. Most things just work, but sometimes it's hard to find out how to do certain things (like removing relations from an entity... _use [set](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#one-to-many-relations)_). I've still been seeing some weird things where there are a _ton_ of processes managing database connections and I haven't quite figured out what I'm doing wrong with that. Otherwise, it works really well.